docs/root/source/decentralized.md

```diff
@@ -14,8 +14,6 @@ A consortium can increase its decentralization (and its resilience) by increasin
 
 There’s no node that has a long-term special position in the cluster. All nodes run the same software and perform the same duties.
 
-MongoDB and RethinkDB have an “admin” user which can’t be deleted and which can make big changes to the database, such as dropping a table. Right now, that’s a big security vulnerability, but we have plans to mitigate it by:
-1. Locking down the admin user as much as possible.
-2. Having all nodes inspect admin-type requests before acting on them. Requests can be checked against an evolving whitelist of allowed actions. Nodes requesting non-allowed requests can be removed from the list of cluster nodes.
+If someone has (or gets) admin access to a node, they can mess with that node (e.g. change or delete data stored on that node), but those changes should remain isolated to that node. The BigchainDB cluster can only be compromised if more than one third of the nodes get compromised. See the [Tendermint documentation](https://tendermint.readthedocs.io/projects/tools/en/master/introduction.html) for more details.
```

Comment:
Even though an attacker cannot manipulate the blockchain but they could possibly throttle the network (DoS) by sending invalid transactions?

Reply:
It's a valid point but it's not related to decentralization IMO.